selfdrive/car/ford/values.py

```diff
@@ -96,13 +101,17 @@ def init_make(self, CP: car.CarParams):
     Request(
       [StdQueries.TESTER_PRESENT_REQUEST, StdQueries.MANUFACTURER_SOFTWARE_VERSION_REQUEST],
       [StdQueries.TESTER_PRESENT_RESPONSE, StdQueries.MANUFACTURER_SOFTWARE_VERSION_RESPONSE],
-      whitelist_ecus=[Ecu.engine],
     ),
     Request(
       [StdQueries.TESTER_PRESENT_REQUEST, StdQueries.MANUFACTURER_SOFTWARE_VERSION_REQUEST],
       [StdQueries.TESTER_PRESENT_RESPONSE, StdQueries.MANUFACTURER_SOFTWARE_VERSION_RESPONSE],
       bus=0,
-      whitelist_ecus=[Ecu.eps, Ecu.abs, Ecu.fwdRadar, Ecu.fwdCamera, Ecu.shiftByWire],
+    ),
+    Request(
+      [StdQueries.TESTER_PRESENT_REQUEST, StdQueries.MANUFACTURER_SOFTWARE_VERSION_REQUEST],
+      [StdQueries.TESTER_PRESENT_RESPONSE, StdQueries.MANUFACTURER_SOFTWARE_VERSION_RESPONSE],
+      bus=0,
+      auxiliary=True,
```

Comment:
Why remove the whitelisting?

Reply:
Also I'm fine with adding more debugging/logging ECUs for data collection and improved fingerprinting, but that should be in a new PR.

Reply:
The F-150 was only getting responses on bus 1, so I had to either add all the ECUs to the whitelist or just remove it. I see now though that it would've been worth investigating this further - `09136c309ba9461d|2023-07-15--18-24-31` in `errorLogMessage`:
- `iso-tp query bad response`
- `iso-tp query timeout after receiving partial response`

Reply:
Adding all ECUs to the new query is the way to go, if you didn't want to affect CAN cars.

Reply:
Oh, I misunderstood what auxiliary meant then. I thought it would query on bus 0 of the auxiliary panda, not "global bus 0" when an auxiliary panda is present?